app/rest/openshift/src/main/java/io/syndesis/openshift/OpenShiftServiceImpl.java

```diff
@@ -210,51 +198,60 @@ private void ensureDeploymentConfig(String name, DeploymentData deploymentData)
             .endVolume()
             .endSpec()
             .endTemplate()
-            .addNewTrigger().withType("ConfigChange").endTrigger()
-// Disabled for now since we trigger the deployment directly
-//            .addNewTrigger().withType("ImageChange")
-//            .withNewImageChangeParams()
-//            // set automatic to 'true' when not performing the deployments on our own
-//            .withAutomatic(false).addToContainerNames(name)
-//            .withNewFrom().withKind("ImageStreamTag").withName(name + ":latest").endFrom()
-//            .endImageChangeParams()
-//            .endTrigger()
+            //.addNewTrigger().withType("ConfigChange").endTrigger()
+            .addNewTrigger().withType("ImageChange")
+            .withNewImageChangeParams()
+            // set automatic to 'true' when not performing the deployments on our own
+            .withAutomatic(true).addToContainerNames(name)
+            .withNewFrom().withKind("ImageStreamTag").withName(name + ":latest").endFrom()
+            .endImageChangeParams()
+            .endTrigger()
             .endSpec()
             .done();
     }
 
 
 
     private boolean removeDeploymentConfig(String projectName) {
+        LOG.debug("Remove Deployment Config {}", projectName);
         return openShiftClient.deploymentConfigs().withName(projectName).delete();
     }
 
     private void ensureBuildConfig(String name, DeploymentData deploymentData, String builderStreamTag, String imageStreamNamespace) {
+        LOG.debug("Create or Replace Build Config {}", name);
+
+        // propagate log level to the build pod
+        List<EnvVar> envVarList = new ArrayList<>();
+        envVarList.add(new EnvVar("MAVEN_OPTS", config.getMavenOptions(), null));
+        if(LOG.isDebugEnabled()){
+            envVarList.add(new EnvVar("BUILD_LOGLEVEL", "5", null));
```

Comment:
Do we want to hardcode the loglevel here or is something worth to consider for configuration ? I think this is good enough for now.